django/contrib/contenttypes/fields.py

```diff
@@ -461,7 +461,7 @@ def get_reverse_path_info(self, filtered_relation=None):
                 to_opts=opts,
                 target_fields=(opts.pk,),
                 join_field=self,
-                m2m=not self.unique,
+                m2m=False,
```

Comment:
Good catch :dart: Looks like it was copied from `ForeignObject.get_reverse_path_info().`